Step02/app/src/main/java/com/kwoolytech/step02/CommonConstant.java

```diff
@@ -0,0 +1,7 @@
+package com.kwoolytech.step02;
+
+
+public class CommonConstant {
+    public static final int BallWidth = 120;
+    public static final int BallHeight = 120;
+}
```

Comment:
@kwoolytech In an android world, you'd better declare size related constants in dimens.xml and get the size in pixel format using API like getDimension()..